src/taipy/core/config/checkers/_scenario_config_checker.py

```diff
@@ -77,3 +92,36 @@ def _check_comparators(self, scenario_config_id: str, scenario_config: ScenarioC
                             f"The value of `{data_node_id}` in {ScenarioConfig._COMPARATOR_KEY} field of ScenarioConfig"
                             f" `{scenario_config_id}` must be populated with a list of Callable values.",
                         )
+
+    def _check_additional_dns_not_overlapping_tasks_dns(self, scenario_config_id: str, scenario_config: ScenarioConfig):
+        data_node_configs = set()
+        for task_config in scenario_config.task_configs:
+            if isinstance(task_config, TaskConfig):
+                input_dn_configs = task_config.input_configs if task_config.input_configs else []
+                output_dn_configs = task_config.output_configs if task_config.output_configs else []
+                data_node_configs.update({*input_dn_configs, *output_dn_configs})
+
+        for additional_data_node_config in scenario_config.additional_data_node_configs:
+            if additional_data_node_config in data_node_configs:
+                self._error(
```

Comment:
well... I don't think it will hurt the execution or anything, does feel weird to have an input or output dn as an additional dn and vice versa though. I doesn't represent correctly what we have in mind for additional dn so I put it as an error. What do you think?

Reply:
Raising an error prevents the user from running its application. Since the pb does not hurt the execution, I would use the WARNING level.